### ISSUE TITLE
Issue with npm install

### ISSUE THREAD

**Issue by safrandsen** [HashLips/generative-art-node#117]
When I run "npm install", I keep getting an error saying that "The package-lock.json file was created with an old version of npm"
npm WARN old lockfile 
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile 
npm WARN old lockfile This is a one-time fix-up, please be patient...
npm WARN old lockfile 

And then I get a series of error codes starting with "npm ERR! code 1". I am not sure what to do - would really appreciate the help!

**Comment by ClaraPrograms** [comment]
+1 same issue - are you also on a mac?

**Comment by safrandsen** [comment]
Yes I am. And I actually figured it out. You need to download an older version of node.js. I downloaded version 14.18.2 and it solved the problem!

**Comment by marielsgtzz** [comment]
> Yes I am. And I actually figured it out. You need to download an older version of node.js. I downloaded version 14.18.2 and it solved the problem!

I am also on mac and THIS SOLVED IT FOR ME THANKS!!
